### ISSUE TITLE
✅ 🐞 Bug Report: Dark mode problem in text editor widget link settings - [ED-13468]

### ISSUE THREAD

**Issue by 79ho3ein** [elementor/elementor#23477]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

Hi,

As mentioned in the title, if the dark mode is enabled, the link settings in the text editor widget will have style problems. Problems in this section :

1. The background color of the fields is dark.
2. The background color of the checkbox is dark.
3. The color of the scroll bar is dark. (Maybe this behavior is normal)
4. When the check box is selected, the screen jumps a bit.
5. When the checkbox is selected, it turns pink when it should be blue.
6. The color of the texts, especially the label fields, is very bright, which hurts the eyes. A little darker would be great.

![Screenshot](https://github.com/elementor/elementor/assets/77163248/998ed4b4-2894-442d-857c-35c8f23847b4)

### Steps to reproduce

1. Make sure the editor is in dark mode.
2. Add the text editor widget.
3. Select a part of the text and click on the link icon.
4. Then click on the settings icon to display the link settings popup.

### Isolating the problem

- [ ] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently following the steps above.

### System Info

<details>
<summary>Click to reveal</summary>

````txt


== Server Environment ==
Operating System: Linux
Software: Apache
MySQL version: MariaDB Server v10.6.14
PHP Version: 8.0.28
PHP Memory Limit: 512M
PHP Max Input Vars: 1000
PHP Max Post Size: 64M
GD Installed: Yes
ZIP Installed: Yes
Write Permissions: All right
Elementor Library: Connected

== WordPress Environment ==
Version: 6.2.2
Site URL: https://domain.com/
Home URL: https://domain.com/
WP Multisite: No
Max Upload Size: 64 MB
Memory limit: 40M
Max Memory limit: 512M
Permalink Structure: /%postname%/
Language: en-US
Timezone: Asia/Tehran
Debug Mode: Inactive

== Theme ==
Name: Hello Elementor
Version: 2.8.1
Author: Elementor Team
Child Theme: No

== User ==
Role: administrator
WP Profile lang: en_US
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/115.0.0.0 Safari/537.36

== Active Plugins ==
Elementor
Version: 3.15.1
Author: Elementor.com

Elementor Pro
	Version: 3.15.0
	Author: Elementor.com

JetEngine
	Version: 3.2.3.1
	Author: crocoblock

Rank Math SEO
	Version: 1.0.120
	Author: Rank Math

WooCommerce
	Version: 7.9.0
	Author: Automattic

WP-Parsidate
	Version: 4.0.2
	Author: WP-Parsi Team

WP Rocket
	Version: 3.14.2
	Author: WP Rocket

WPS Hide Login
	Version: 1.9.8
	Author: WPServeur, NicolasKulka, wpformation
== Elements Usage ==

archive : 2
	container : 4
	heading : 2
	loop-grid : 2
	text-editor : 2
footer : 1
	container : 6
	form : 1
	heading : 5
	icon-list : 2
	social-icons : 1
	text-editor : 1
header : 1
	button : 1
	container : 2
	nav-menu : 2
	theme-site-logo : 2
	woocommerce-menu-cart : 2
page : 1
	container : 1
	countdown : 1
	heading : 2
popup : 4
	container : 6
	countdown : 1
	heading : 4
	lottie : 2
	nav-menu : 1
wp-page : 5
	button : 5
	container : 22
	counter : 3
	divider : 6
	form : 2
	heading : 14
	icon : 3
	icon-box : 13
	icon-list : 5
	image : 4
	loop-grid : 2
	social-icons : 1
	text-editor : 3
	toggle : 4
	woocommerce-breadcrumb : 1
single-post : 2
	button : 1
	container : 11
	divider : 6
	form : 1
	heading : 9
	icon-list : 20
	image : 2
	image-box : 1
	jet-engine-data-store-button : 1
	jet-listing-dynamic-field : 1
	loop-grid : 1
	post-comments : 1
	share-buttons : 2
	table-of-contents : 1
	text-editor : 1
	theme-post-content : 1
	woocommerce-breadcrumb : 2
loop-item : 3
	container : 6
	heading : 1
	icon-list : 2
	image : 2
	progress : 1
	theme-post-featured-image : 1
	theme-post-title : 2
error-404 : 1
	button : 1
	heading : 1
	lottie : 1
== Settings ==

cpt_support: post, page
allow_tracking: yes
unfiltered_files_upload: 1
font_display: swap
== Features ==
Custom Fonts: 2
Custom Icons: 0

== Integrations ==

recaptcha_v3: Active
woocommerce: Active
== Elementor Experiments ==
Optimized DOM Output: Active
Improved Asset Loading: Active
Improved CSS Loading: Active
Inline Font Icons: Inactive
Additional Custom Breakpoints: Inactive
admin_menu_rearrangement: Inactive by default
Flexbox Container: Active
Upgrade Swiper Library: Active
Grid Container: Inactive
Default to New Theme Builder: Inactive
Hello Theme Header & Footer: Inactive
Editor Top Bar: Active
Landing Pages: Inactive
Nested Elements: Active
Lazy Load Background Images: Inactive
Global Style Guide: Active
Page Transitions: Inactive
Notes: Inactive
Loop: Active
Form Submissions: Active
Scroll Snap: Inactive
Menu: Active
Taxonomy Filter: Inactive by default

== Log ==

Log: showing 20 of 392023-06-27 11:49:19 [info] elementor::elementor_updater Started
2023-06-27 11:49:19 [info] elementor/Upgrades - _on_each_version Start
2023-06-27 11:49:20 [info] elementor/Upgrades - _on_each_version Finished
2023-06-27 11:49:20 [info] Elementor data updater process has been completed. [array (
'plugin' => 'elementor',
'from' => '3.14.0',
'to' => '3.14.1',
)]
2023-06-27 11:49:55 [info] elementor-pro::elementor_pro_updater Started
2023-06-27 11:49:56 [info] Elementor Pro/Upgrades - _on_each_version Start
2023-06-27 11:49:56 [info] Elementor Pro/Upgrades - _on_each_version Finished
2023-06-27 11:49:56 [info] Elementor data updater process has been completed. [array (
'plugin' => 'Elementor Pro',
'from' => '3.14.0',
'to' => '3.14.1',
)]
2023-08-03 17:06:17 [info] elementor::elementor_updater Started
2023-08-03 17:06:17 [info] elementor/Upgrades - _on_each_version Start
2023-08-03 17:06:17 [info] Elementor data updater process has been queued. [array (
'plugin' => 'elementor',
'from' => '3.14.1',
'to' => '3.15.1',
)]
2023-08-03 17:06:18 [info] elementor/Upgrades - _on_each_version Finished
2023-08-03 17:06:18 [info] Elementor data updater process has been completed. [array (
'plugin' => 'elementor',
'from' => '3.14.1',
'to' => '3.15.1',
)]
2023-08-03 17:06:18 [info] Elementor data updater process has been queued. [array (
'plugin' => 'elementor',
'from' => '3.14.1',
'to' => '3.15.1',
)]
2023-08-03 17:10:04 [info] elementor-pro::elementor_pro_updater Started
2023-08-03 17:10:04 [info] Elementor Pro/Upgrades - _on_each_version Start
2023-08-03 17:10:04 [info] Elementor Pro/Upgrades - _on_each_version Finished
2023-08-03 17:10:04 [info] Elementor data updater process has been completed. [array (
'plugin' => 'Elementor Pro',
'from' => '3.14.1',
'to' => '3.15.0',
)]
2023-08-03 17:10:04 [info] Elementor data updater process has been queued. [array (
'plugin' => 'Elementor Pro',
'from' => '3.14.1',
'to' => '3.15.0',
)]
2023-08-03 17:10:05 [info] Elementor data updater process has been queued. [array (
'plugin' => 'Elementor Pro',
'from' => '3.14.1',
'to' => '3.15.0',
)]

PHP: showing 2 of 2PHP: 2023-05-23 16:16:04 [warning X 1][../wp-content/plugins/elementor/includes/managers/image.php::108] Trying to access array offset on value of type bool [array (
'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-05-24 22:57:21 [warning X 737][../wp-content/plugins/elementor-pro/modules/woocommerce/module.php::381] Undefined array key 1 [array (
'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]

JS: showing 2 of 2JS: 2023-06-11 11:52:58 [error X 2][https://avangwebsite.ir/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.13.4:3:1097282] Cannot read properties of null (reading 'getBoundingClientRect')
JS: 2023-06-19 12:31:58 [error X 12][https://avangwebsite.ir/wp-content/plugins/elementor/assets/js/7fada1a61d72da6d46ab.bundle.min.js:2:7826] Cannot read properties of null (reading 'dataset')

== Elementor - Compatibility Tag ==

Elementor Pro: Compatible
== Elementor Pro - Compatibility Tag ==

````

</details>

**Comment by nicholaszein** [comment]
Hello @79ho3ein ! 

Thank you for submitting your issue! 🙏 

🏷️ We've labeled this issue and our team will take a look at it. If there is indeed a problem, they'll work on fixing it. We can't say exactly when it'll be sorted out, but we kindly ask for your patience. Thanks for understanding!

Best regards

**Comment by nicholaszein** [comment]
Hey @79ho3ein!

I know I'm little late, but I'm just passing to let you know that...

### 📢 We're happy to announce that the issue you raised was resolved in Elementor Pro v3.19.0! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
